### DISCUSSION TITLE
Tailwind CSS with .Net Core MVC

### DISCUSSION THREAD

**Discussion by AAAi7** [tailwindlabs/tailwindcss#3398] [Help]
I have been going through all kinds of tutorials they are either just straight up not useful or using previous versions of VS.
How do you install tailwind css on a MVC project?
I wish they had published the lib for everyone to copy and paste in their resource file and use it freely.

**Comment by rus19023** [comment]
On a freshly created project, I had tailwind @apply directives working, but when I installed the .net 5 frameworks needed to make the program run, it broke the css compilation somewhere. I am new to C# and .net, so I haven't been able to figure out what breaks it.
